_posts/2020-06-02-postdoc-position.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: Job announcement
+subtitle: 3-year postdoc postion in glacier modeling at the University of Bremen
+author: Matthias Dusch
```

Comment:
Put Ben here, otherwise its not clear I think

Reply:
true, done!